src/anomalib/callbacks/metrics.py

```diff
@@ -88,14 +88,23 @@ def setup(
             )
 
         if isinstance(pl_module, AnomalyModule):
-            pl_module.image_metrics = create_metric_collection(image_metric_names, "image_")
-            if hasattr(pl_module, "pixel_metrics"):  # incase metrics are loaded from model checkpoint
+            pl_module.image_metrics = create_metric_collection(
+                image_metric_names,
+                "image_",
+            )
+            if hasattr(
+                pl_module,
+                "pixel_metrics",
+            ):  # incase metrics are loaded from model checkpoint
```

Comment:
there is a quite a bit of diff in the PR that just re-formats the code, which is out-of-scope of the pr.

I believe this change is not needed. could this be due to some version mismatch in formatters such as black or ruff?

Reply:
Definitely as suspected was a mismatch with Ruff I believe.
It has been addressed and now it is easier to navigate the changes.